addons/mail/models/mail_channel.py

```diff
@@ -351,7 +351,6 @@ def _extract_moderation_values(self, message_type, **kwargs):
                 moderation_status = 'pending_moderation'
         return moderation_status, email
 
-    @api.returns('mail.message', lambda value: value.id)
     def message_post(self, message_type='notification', **kwargs):
```

Comment:
The calls to `message_post` from the JS must be adapted, including the mocks of `message_post` in JS tests.